test/1_stdlib/PrintFloat.swift

```diff
@@ -191,17 +209,57 @@ PrintTests.test("Printable") {
   expectDebugPrinted("1.24999998e+17", asFloat32(125000000000000000.0))
   expectDebugPrinted("1.25", asFloat32(1.25))
   expectDebugPrinted("1.24999997e-05", asFloat32(0.0000125))
+  expectDebugPrinted("inf", Float.infinity)
+  expectDebugPrinted("-inf", -Float.infinity)
+  expectDebugPrinted("nan", Float.nan)
+  expectDebugPrinted("-nan", -Float.nan)
+  expectDebugPrinted("nan(0xffff)", Float(nan: 65535, signaling: false))
+  expectDebugPrinted("-nan(0xffff)", -Float(nan: 65535, signaling: false))
+  expectDebugPrinted("nan(0x1fffff)", Float(bitPattern: 0x7fff_ffff))
+  expectDebugPrinted("nan(0x1fffff)", Float(bitPattern: 0x7fdf_ffff))
```

Comment:
Actually, current implementation drops MSB of payload,
because that bit is reserved for placeholder payload of signaling NaN.
I'm not sure whether this behavior is reasonable or not. @stephentyrone ?